### ISSUE TITLE
when I run  synthesizer_train.py , got Error

### ISSUE THREAD

**Issue by peter05010402** [CorentinJ/Real-Time-Voice-Cloning#239]
Exiting due to exception: indices[5,21] = 90 is not in [0, 66)
         [[Node: Tacotron_model/inference/embedding_lookup = GatherV2[Taxis=DT_INT32, Tindices=DT_INT32, Tparams=DT_FLOAT, _class=["loc:@Tacot.../ApplyAdam"], _device="/job:localhost/replica:0/task:0/device:CPU:0"](Tacotron_model/inference/inputs_embedding/read, Tacotron_model/Reshape, Tacotron_model/inference/embedding_lookup/axis)]]

Caused by op 'Tacotron_model/inference/embedding_lookup', defined at:
  File "synthesizer_train.py", line 55, in <module>
    tacotron_train(args, log_dir, hparams)
  File "/mnt1/Real-Time-Voice-Cloning/synthesizer/train.py", line 392, in tacotron_train
    return train(log_dir, args, hparams)
  File "/mnt1/Real-Time-Voice-Cloning/synthesizer/train.py", line 148, in train
    model, stats = model_train_mode(args, feeder, hparams, global_step)
  File "/mnt1/Real-Time-Voice-Cloning/synthesizer/train.py", line 91, in model_train_mode
    is_training=True, split_infos=feeder.split_infos)
  File "/mnt1/Real-Time-Voice-Cloning/synthesizer/models/tacotron.py", line 137, in initialize
    embedded_inputs = tf.nn.embedding_lookup(self.embedding_table, tower_inputs[i])
  File "/home/tensorflow-GPU/lib/python3.6/site-packages/tensorflow/python/ops/embedding_ops.py", line 308, in embedding_lookup
    transform_fn=None)
  File "/home/tensorflow-GPU/lib/python3.6/site-packages/tensorflow/python/ops/embedding_ops.py", line 131, in _embedding_lookup_and_transform
    result = _clip(array_ops.gather(params[0], ids, name=name),
  File "/home/tensorflow-GPU/lib/python3.6/site-packages/tensorflow/python/ops/array_ops.py", line 2736, in gather
    return gen_array_ops.gather_v2(params, indices, axis, name=name)
  File "/home/tensorflow-GPU/lib/python3.6/site-packages/tensorflow/python/ops/gen_array_ops.py", line 3065, in gather_v2
    "GatherV2", params=params, indices=indices, axis=axis, name=name)
  File "/home/tensorflow-GPU/lib/python3.6/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/home/tensorflow-GPU/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3392, in create_op
    op_def=op_def)
  File "/home/tensorflow-GPU/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 1718, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

InvalidArgumentError (see above for traceback): indices[5,21] = 90 is not in [0, 66)
         [[Node: Tacotron_model/inference/embedding_lookup = GatherV2[Taxis=DT_INT32, Tindices=DT_INT32, Tparams=DT_FLOAT, _class=["loc:@Tacot.../ApplyAdam"], _device="/job:localhost/replica:0/task:0/device:CPU:0"](Tacotron_model/inference/inputs_embedding/read, Tacotron_model/Reshape, Tacotron_model/inference/embedding_lookup/axis)]]
